pkg/manager/member/tikv_member_manager.go

```diff
@@ -613,7 +615,12 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 	podSpec.Volumes = append(vols, baseTiKVSpec.AdditionalVolumes()...)
 	podSpec.SecurityContext = podSecurityContext
 	podSpec.InitContainers = append(initContainers, baseTiKVSpec.InitContainers()...)
-	podSpec.Containers = append(containers, baseTiKVSpec.AdditionalContainers()...)
+
+	podSpec.Containers, err = MergePatchContainers(containers, baseTiKVSpec.AdditionalContainers())
```

Comment:
Please also update the code for TiDB Monitor, tidbngmonitoring, DM master, DM worker, Discovery, TiCDC, and Pump.

Reply:
ok